### PR TITLE
Pull Request for Code Review

### PR DESCRIPTION
**Pull request by frederico-miranda**
This is a pull request opened for receiving a code review. There isn't a link for a live version of the page because _raw.githack.com_ is failing frequently today.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+<html lang="en-US">
+  <head>
+    <meta charset="UTF-8"/>
+    <title>Apple</title>
+    <link rel="stylesheet" href="styles/normalize.css" />
+    <link rel="stylesheet" href="styles/main.css" />
+  </head>
+  <body>
+    <main>
+      <nav class="main-navigation">
```
Comment (suretrust) [inline index.html]: The `<main>` tag should not have the `<nav>` or `<footer>` tag in it. It should be arranged as shown below:
`<nav>Navigation bar</nav>`
`<main>Main content</main>`
`<footer>Footer section</footer>`

File: index.html [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+      </article>
+
+      <aside class="advertisements">
+        <h2>Advertisements</h2>
+        <section class="advertisements-entry iphone-5s-advertisement">
+          <a href="">
+            <hgroup>
+              <h2>iPhone 5S</h2>
+              <h3>Forward thinking.</h3>
+            </hgroup>
```
Comment (suretrust) [inline index.html]: Lovely using `<hgroup>`. However, the `<hgroup>` element has been removed from the HTML5 (W3C) specification, though it still is in the WHATWG version of HTML. Proof [is under usage notes here](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/hgroup).

The HTML5 (W3C) specification provides advice on how to mark up [Subheadings, subtitles, alternative titles and taglines without using `<hgroup>`](https://www.w3.org/TR/html52/common-idioms-without-dedicated-elements.html#common-idioms-without-dedicated-elements).

Kindly remove the `<hgroup>` tag.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+          </a>
+        </section>
+
+        <section class="advertisements-entry iphone-5c-advertisement">
+          <a href="">
+            <hgroup>
+              <h2>iPhone 5C</h2>
+              <h3>For the colorful.</h3>
+            </hgroup>
```
Comment (suretrust) [inline index.html]: Kindly remove `<hgroup>` tag.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+          </a>
+        </section>
+
+        <section class="advertisements-entry mac-advertisement">
+          <a href="">
+            <hgroup>
+              <h2>Happy Birthday, Mac.</h2>
+              <h3>Explore 30 years of innovation on the Mac.</h3>
+            </hgroup>
```
Comment (suretrust) [inline index.html]: Kindly remove `<hgroup>` tag.

File: index.html [hunk 5]
```diff
@@ -0,0 +1,101 @@
...
+          </a>
+        </section>
+
+        <section class="advertisements-entry supplier-responsibility-advertisement">
+          <a href="">
+            <hgroup>
+              <h2>Supplier Responsibility</h2>
+              <h3>The latest results of our worldwide efforts.</h3>
+            </hgroup>
```
Comment (suretrust) [inline index.html]: Kindly remove `<hgroup>` tag.

File: index.html [hunk 6]
```diff
@@ -0,0 +1,101 @@
...
+    <meta charset="UTF-8"/>
+    <title>Apple</title>
+    <link rel="stylesheet" href="styles/normalize.css" />
+    <link rel="stylesheet" href="styles/main.css" />
+  </head>
+  <body>
+    <main>
+      <nav class="main-navigation">
+        <ul>
+          <li><a href=""><span class="apple-icon"></span> Apple</a></li>
```
Comment (suretrust) [inline index.html]: The Apple icon does not have `Apple` as content in the original webpage.
\
apple

File: index.html [hunk 7]
```diff
@@ -0,0 +1,101 @@
...
+        <section class="advertisements-entry iphone-5c-advertisement">
+          <a href="">
+            <hgroup>
+              <h2>iPhone 5C</h2>
+              <h3>For the colorful.</h3>
+            </hgroup>
+          </a>
+        </section>
+
+        <section class="advertisements-entry mac-advertisement">
```
Comment (suretrust) [inline index.html]: Lovely using the `<section>` tag. :+1:

**Comment by frederico-miranda** [comment]
@suretrust 

I've implemented the changes you have pointed out. I've also restyle the navigation bar so that it resembles the original one more closely. Do I need to ask for a new code review on the Microverse dashboard or is simply calling you here enough?

**Comment by ebukaume** [comment]
## Approved! :champagne: 

Hi @frederico-miranda,

Well done. You clone now looks like the real deal. **You can now merge this project**

Happy Coding :v: 
**Ebuka Umeokonkwo**
 [Github](https://github.com/ebukaume) | [Linkedin](https://www.linkedin.com/in/ebukaume) | [Slack](https://app.slack.com/team/UFDKXE68L) | [Twitter](https://twitter.com/ebukaume)

**Pull request merged**